recipes/r-repurrrsive/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set version = '1.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-repurrrsive
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/repurrrsive_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/repurrrsive/repurrrsive_{{ version }}.tar.gz
+  sha256: a9c109889293a4f2ba1defc198f2c1f2590feb3744105e97f4926808f1717088
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-tibble
+  run:
+    - r-base
+    - r-tibble
+
+test:
+  commands:
+    - $R -e "library('repurrrsive')"           # [not win]
+    - "\"%R%\" -e \"library('repurrrsive')\""  # [win]
+
+about:
+  home: https://jennybc.github.io/repurrrsive/, https://github.com/jennybc/repurrrsive
+  license: CC0-1.0
+  summary: Recursive lists in the form of R objects, 'JSON', and 'XML', for use in teaching and
+    examples. Examples include color palettes, Game of Thrones characters, 'GitHub'
+    users and repositories, music collections, and entities from the Star Wars universe.
+    Data from the 'gapminder' package is also included, as a simple data frame and in
+    nested and split forms.
+  license_family: CC
+
```

Need to package license.

```suggestion
  license_file:
    - LICENSE

```